eslint.config.mjs

```diff
@@ -0,0 +1,23 @@
+import config from '@iobroker/eslint-config';
+
+// disable temporary the rule 'jsdoc/require-param' and enable 'jsdoc/require-jsdoc'
+config.forEach(rule => {
```

Comment:
In backend adapters for sure warnings don't hurt and stuff can be fixed in the long run. So no need to disable these rules. However I have fixed all warnings